generators/server/templates/_pom.xml

```diff
@@ -1200,7 +1209,7 @@
                 <!-- log configuration -->
                 <logback.loglevel>INFO</logback.loglevel>
                 <!-- default Spring profiles -->
-                <spring.profiles.active>prod</spring.profiles.active>
+                <spring.profiles.active>prod${profile.no-liquibase}</spring.profiles.active>
```

Comment:
please add `${profile.no-swagger}` as well coz, people can enable swagger for production in application.yml and then can choose to disable temporarily using the profile switch